rtgui/locallab.cc

```diff
@@ -331,15 +332,27 @@ void Locallab::read(const rtengine::procparams::ProcParams* pp, const ParamsEdit
         }
 
         r.transit = pp->locallab.spots.at(i).transit;
-        r.transitweak = pp->locallab.spots.at(i).transitweak;
-        r.transitgrad = pp->locallab.spots.at(i).transitgrad;
+   //     if(showsettings != 2) {
+            r.transitweak = pp->locallab.spots.at(i).transitweak;
+            r.transitgrad = pp->locallab.spots.at(i).transitgrad;
+    //    } else {
+    //        r.transitweak = 1.;
+    //        r.transitgrad = 0.;
+    //    }
```

Comment:
Do you want to keep the commented lines?

Reply:
I remove